pkg/controller/bypass/bypass_controller.go

```diff
@@ -78,15 +80,26 @@ func StartByPassController(client kubernetes.Interface) error {
 
 			log.Infof("%s/%s: ADDED", pod.GetNamespace(), pod.GetName())
 			enableSidecar, _ := checkSidecar(client, pod)
-			if !enableSidecar {
-				log.Info("do not need add iptables rules, pod is not managed by sidecar")
+			enableKmesh, _ := checkKmesh(client, pod)
+			if !enableSidecar && !enableKmesh {
+				log.Info("do not need process, pod is not managed by sidecar or kmesh")
 				return
 			}
 
 			nspath, _ := getnspath(pod)
-			if err := addIptables(nspath); err != nil {
-				log.Errorf("failed to add iptables rules for %s: %v", nspath, err)
-				return
+
+			if enableSidecar {
+				if err := addIptables(nspath); err != nil {
+					log.Errorf("failed to add iptables rules for %s: %v", nspath, err)
+					return
+				}
+			}
+			if enableKmesh {
+				log.Infof("enableKmesh!,, bypass kmesh,%s", nspath)
```

Comment:
```suggestion
				log.Infof("enableKmesh!, bypass kmesh,%s", nspath)
```
enableKmesh but bypass kmesh.Is kmesh used or not used?